types/postcss-import/postcss-import-tests.ts

```diff
@@ -0,0 +1,23 @@
+import * as postcss from 'postcss';
```

Comment:
Since this is a callable or constructable entity, it isn't legal in ES2015 to import and call/construct this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either

```ts
import foo = require('foo')
```

or the following with `esModuleInterop` turned on

```ts
import foo from 'foo';
```